src/main.js

```diff
@@ -73,7 +83,10 @@ async function historicLandCover(browserPosition) {
 function getLocation() {
   if (navigator.geolocation) {
     navigator.geolocation.getCurrentPosition(
-      historicLandCover,
+      historicLandCoverBrowser,
+      // () => historicLandCover([-89.43, 43.06], 'Wisconsin'),
+      // () => historicLandCover([-88.33, 40.91], 'Illinois'),
+      // () => historicLandCover([-84.75, 42.38], 'Michigan'),
```

Comment:
I used these lines for testing to make testing across state lines easier. I couldn't test Illinois though, I got a 403 Forbidden from the arcgis proxy:
> The origin "http://localhost:8000" was not whitelisted by the operator of this proxy.